scripts/install_redis.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 # This script is meant to be run at buildtime.
 
-# These updates must be run, or else old redis could be installed (like version 5.0.7)
```

Comment:
There is a legacy script to install redis to ubuntu, probably left over from v1. I've deleted that script, and renamed `install_redis_centos.sh` to `install_redis.sh`. Therefore the diff is a bit confusing.